lib/storage/src/content_manager/consensus/persistent.rs

```diff
@@ -150,6 +154,25 @@ impl Persistent {
         self.save()
     }
 
+    pub fn update_peer_metadata(
+        &mut self,
+        peer_id: PeerId,
+        metadata: PeerMetadata,
+    ) -> Result<(), StorageError> {
+        if let Some(prev_metadata) = self
+            .peer_metadata_by_id
+            .write()
+            .insert(peer_id, metadata.clone())
+        {
+            log::warn!(
+                "Replaced metadata of peer {peer_id} from {prev_metadata:?} to {metadata:?}"
+            );
```

Comment:
`log::info` maybe? I guess metadata won't be updated _too_ often (as we will only use it for Qdrant version; _for now_), but `log::warn` just seems too verbose for a pretty regular, expected condition.

Reply:
Better, I agree.